uploader.json

```diff
@@ -3539,8 +3549,8 @@
         "ceph_health_detail": {
             "host": [
                 {
-                    "archive_file_name": "/insights_commands/ceph_health_detail_-f_json-pretty",
-                    "command": "/usr/bin/ceph health detail -f json-pretty",
+                    "archive_file_name": "/insights_commands/ceph_health_detail_-f_json",
+                    "command": "/usr/bin/ceph health detail -f json",
```

Comment:
@psachin please remove all change after line 54, this section of the JSON does not change.

Reply:
@psachin after discussion with the insights-client team these changes may be OK, hold off on any changes.

Reply:
Ack